spec/requests/api/graphql/mutations/add_asset_to_consignment_submission_spec.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/gravity_helper'
+
+describe 'addAssetToConsignmentSubmission mutation' do
+  let(:user) { Fabricate(:user, gravity_user_id: 'userid') }
+  let(:submission) { Fabricate(:submission, user: user) }
+
+  let(:token) do
+    payload = { aud: 'gravity', sub: user.gravity_user_id, roles: 'user' }
+    JWT.encode(payload, Convection.config.jwt_secret)
+  end
+
+  let(:headers) { { 'Authorization' => "Bearer #{token}" } }
+
+  let(:mutation_inputs) do
+    "{ clientMutationId: \"test\", submissionID: #{
+      submission.id
+    }, geminiToken: \"gemini-token-hash\" }"
```

Comment:
Go home prettier Ruby you're drunk. 😭

Reply:
I wonder if this is due to the outer `"`, instead of a `'`? Or in ruby is the `"` required for interpolation like this?

Reply:
So this is about interp like you said. Maybe there's a better way for me to format it, but I couldn't find it. 🤷‍♂